docs/products/kafka/index.rst

```diff
@@ -0,0 +1,48 @@
+Aiven for Apache Kafka
+======================
+
+What is Aiven for Apache Kafka?
+-------------------------------
+
+Aiven for Apache Kafka is a fully managed **distributed data streaming platform**, deployable in the cloud of your choice. Apache Kafka is an open source data streaming platform, ideal for event-driven applications, near-real-time data transfer and pipelines, stream analytics, and many more applications where a lot of data needs to move between applications in a speedy manner.
+
+Kafka stores a potentially large number of small data payloads, usually for a limited period of time. The storage is organised into "topics" and "partitions" so that many data streams can be handled at once, regardless of how many producers or consumers you need to work with.
```

Comment:
Should we state that are messages or events that Kafka stores?

> Kafka stores a potentially large number of **messages** having small data payload

Reply:
For Kafka I believe the correct term would be records. If you think knowing what it is called would help to understand the shape of the data, we can add it. But I moved the records mention to later because I was just on storage at this point.